
9/7/2019 11:53:54 AM
------------------------------------------------------------------------------------------------
Controller Name :- Account
Action Method Name :- Login
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:53:56 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:53:56 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:53:57 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:53:58 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:53:59 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:53:59 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:01 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:01 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:02 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:03 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:03 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:04 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:04 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:05 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:06 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:06 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:07 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:07 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:08 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:08 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:09 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:09 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:10 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:10 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:11 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:11 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:12 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:12 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:13 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:13 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:13 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:14 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:14 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:15 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:16 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:16 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:17 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:17 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:23 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:23 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:23 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:24 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:25 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:25 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:26 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:27 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:28 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:28 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:29 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:29 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:29 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:30 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:31 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:31 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:32 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:32 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:33 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:54:33 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:04 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:04 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:04 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:05 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:06 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:06 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:06 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:07 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:07 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:08 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:08 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:09 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:09 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:09 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:10 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:11 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:11 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:12 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:12 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:55:12 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:13 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:13 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:14 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:14 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:15 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:15 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:16 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:16 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:17 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:17 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:17 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:18 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:18 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:18 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:19 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:19 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:20 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:20 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:21 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 11:56:21 AM
------------------------------------------------------------------------------------------------
Controller Name :- Errorview
Action Method Name :- Error
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}Object reference not set to an instance of an object.
Source ---
{0}Protium.Web
StackTrace ---
{0}   at Protium.Web.Filters.AuditFilter.OnActionExecuting(ActionExecutingContext context) in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Filters\AuditFilter.cs:line 45
   at Microsoft.AspNetCore.Mvc.Filters.ActionFilterAttribute.OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}Void OnActionExecuting(Microsoft.AspNetCore.Mvc.Filters.ActionExecutingContext)


9/7/2019 6:33:45 PM
------------------------------------------------------------------------------------------------
Controller Name :- Requisition
Action Method Name :- Index
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}A second operation started on this context before a previous operation completed. This is usually caused by different threads using the same instance of DbContext, however instance members are not guaranteed to be thread safe. This could also be caused by a nested query being evaluated on the client, if this is the case rewrite the query avoiding nested invocations.
Source ---
{0}Microsoft.EntityFrameworkCore
StackTrace ---
{0}   at Microsoft.EntityFrameworkCore.Internal.ConcurrencyDetector.EnterCriticalSection()
   at Microsoft.EntityFrameworkCore.Internal.ConcurrencyDetector.EnterCriticalSectionAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.Aggregate_[TSource,TAccumulate,TResult](IAsyncEnumerable`1 source, TAccumulate seed, Func`3 accumulator, Func`2 resultSelector, CancellationToken cancellationToken) in D:\a\1\s\Ix.NET\Source\System.Interactive.Async\Aggregate.cs:line 120
   at Protium.Repository.PermissionRepo.RequisitionRepository.GetRequisitionsByUserAsync() in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Repository\Repos\RequisitionRepository.cs:line 67
   at Protium.Web.Controllers.RequisitionController.Index() in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Controllers\RequisitionController.cs:line 61
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}System.IDisposable EnterCriticalSection()


9/7/2019 6:34:55 PM
------------------------------------------------------------------------------------------------
Controller Name :- Requisition
Action Method Name :- Index
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}A second operation started on this context before a previous operation completed. This is usually caused by different threads using the same instance of DbContext, however instance members are not guaranteed to be thread safe. This could also be caused by a nested query being evaluated on the client, if this is the case rewrite the query avoiding nested invocations.
Source ---
{0}Microsoft.EntityFrameworkCore
StackTrace ---
{0}   at Microsoft.EntityFrameworkCore.Internal.ConcurrencyDetector.EnterCriticalSection()
   at Microsoft.EntityFrameworkCore.Internal.ConcurrencyDetector.EnterCriticalSectionAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.Aggregate_[TSource,TAccumulate,TResult](IAsyncEnumerable`1 source, TAccumulate seed, Func`3 accumulator, Func`2 resultSelector, CancellationToken cancellationToken) in D:\a\1\s\Ix.NET\Source\System.Interactive.Async\Aggregate.cs:line 120
   at Protium.Repository.PermissionRepo.RequisitionRepository.GetRequisitionsByUserAsync() in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Repository\Repos\RequisitionRepository.cs:line 67
   at Protium.Web.Controllers.RequisitionController.Index() in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Controllers\RequisitionController.cs:line 61
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}System.IDisposable EnterCriticalSection()


9/7/2019 6:35:15 PM
------------------------------------------------------------------------------------------------
Controller Name :- ItemCategory
Action Method Name :- Index
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}A second operation started on this context before a previous operation completed. This is usually caused by different threads using the same instance of DbContext, however instance members are not guaranteed to be thread safe. This could also be caused by a nested query being evaluated on the client, if this is the case rewrite the query avoiding nested invocations.
Source ---
{0}Microsoft.EntityFrameworkCore
StackTrace ---
{0}   at Microsoft.EntityFrameworkCore.Internal.ConcurrencyDetector.EnterCriticalSection()
   at Microsoft.EntityFrameworkCore.Internal.ConcurrencyDetector.EnterCriticalSectionAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.Aggregate_[TSource,TAccumulate,TResult](IAsyncEnumerable`1 source, TAccumulate seed, Func`3 accumulator, Func`2 resultSelector, CancellationToken cancellationToken) in D:\a\1\s\Ix.NET\Source\System.Interactive.Async\Aggregate.cs:line 120
   at Protium.Repository.PermissionRepo.ItemCategoryRepository.GetItemCategoryAsync() in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Repository\Repos\ItemCategoryRepository.cs:line 38
   at Protium.Web.Controllers.ItemCategoryController.Index() in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Controllers\ItemCategoryController.cs:line 29
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}System.IDisposable EnterCriticalSection()


9/7/2019 6:36:16 PM
------------------------------------------------------------------------------------------------
Controller Name :- Requisition
Action Method Name :- Index
------------------------------------------------------------------------------------------------
Microsoft.AspNetCore.Mvc.Filters.ExceptionContext
Message ---
{0}A second operation started on this context before a previous operation completed. This is usually caused by different threads using the same instance of DbContext, however instance members are not guaranteed to be thread safe. This could also be caused by a nested query being evaluated on the client, if this is the case rewrite the query avoiding nested invocations.
Source ---
{0}Microsoft.EntityFrameworkCore
StackTrace ---
{0}   at Microsoft.EntityFrameworkCore.Internal.ConcurrencyDetector.EnterCriticalSection()
   at Microsoft.EntityFrameworkCore.Internal.ConcurrencyDetector.EnterCriticalSectionAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.Aggregate_[TSource,TAccumulate,TResult](IAsyncEnumerable`1 source, TAccumulate seed, Func`3 accumulator, Func`2 resultSelector, CancellationToken cancellationToken) in D:\a\1\s\Ix.NET\Source\System.Interactive.Async\Aggregate.cs:line 120
   at Protium.Repository.PermissionRepo.RequisitionRepository.GetRequisitionsByUserAsync() in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Repository\Repos\RequisitionRepository.cs:line 67
   at Protium.Web.Controllers.RequisitionController.Index() in C:\Users\User\Source\Repos\CyberspaceRequisition\Protium.Web\Controllers\RequisitionController.cs:line 61
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()
TargetSite ---
{0}System.IDisposable EnterCriticalSection()

